src/app-designer-prereqs.rst

```diff
@@ -243,6 +243,16 @@ For Windows
   #. Click on :guilabel:`OK` and exit all of the windows.
   #. Verify that you have made the change by closing all :program:`PowerShell` or :program:`cmd` windows and open a new one (so it picks up the change), and type
 
+.. Outreachy 2024 Starter Contribution Guide
```

Comment:
I am not sure we want to have documentation that is specific to Outreachy and 2024.